### PR TITLE
Add help doc for msync

### PR DESCRIPTION
**Pull request by bbriggs**

### CODE REVIEW

File: docs/msync.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## TheVoxFox Usage: !msync
+
+TheVoxFox accept a small range of arguments that you can use to retrieve modulesync version statues or generate reports.
```
Comment (bastelfreak) [inline docs/msync.md]: accepts? (I've no idea)

File: docs/msync.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Using `!msync` with no args will retrieve the latest semver tag from our modulesync_config repo.
```
Comment (bastelfreak) [inline docs/msync.md:7]: should we link to https://github.com/voxpupuli/modulesync_config (if somebody is new to our infra, me maybe doesn't know the link?)

File: docs/msync.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+### Args
+
+#### getver
+
+Query the last applied modudlesync config version applied to a specific repo.
```
Comment (bastelfreak) [inline docs/msync.md]: modudlesync typo

**Review by bastelfreak — CHANGES_REQUESTED** [review]

File: docs/msync.md (hunk 1 above)
Comment (bbriggs) [reply docs/msync.md]: Dang it, I'm the English speaker here. I'm supposed to be the one getting this part right :(

File: docs/msync.md (hunk 2 above)
Comment (bbriggs) [reply docs/msync.md:7]: Yeah, good idea

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**